packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/cli/cli.controller.js

```diff
@@ -0,0 +1,15 @@
+import { CLI_INSTALL_CODE } from './cli.constants';
+
+export default class AIDashboardCliCtrl {
+  /* @ngInject */
+  constructor($translate, $timeout, coreConfig, atInternet) {
+    this.$translate = $translate;
+    this.$timeout = $timeout;
+    this.coreConfig = coreConfig;
```

Comment:
**nitpick:** Remove this unused dependencies
```suggestion
```

Reply:
Removed: https://github.com/ovh/manager/pull/8213/commits/9f4aaae0907eaded7c1a7374a222516dd3eb6044